packages/mongoose-store/src/models/file.ts

```diff
@@ -0,0 +1,30 @@
+'use strict';
```

Comment:
Let's remove this schemas as we do not even want to support them initially in raincatcher